packages/deployer/test/errors.js

```diff
@@ -213,8 +213,8 @@ describe("Error cases", function() {
       assert.fail();
     } catch (err) {
       assert(err.message.includes("Loops"));
-      assert(err.message.includes("code couldn't be stored"));
-      assert(err.message.includes("check your gas limit"));
+      assert(err.message.includes("ran out of gas"));
+      assert(err.message.includes("network config or deployment parameters"));
```

Comment:
I assume this is just changes in error messages?

Reply:
there was a regression that's been fixed in this release. I'll change this commit to use the original (now fixed in this release) output we were looking for https://github.com/trufflesuite/truffle/blame/be86b8eed120af9dca7d2ea1ee751a54ded200fc/packages/deployer/test/errors.js#L215-L218.

Reply:
Oh yeah that whole thing... cool.